api/src/indigo_molecule.cpp

```diff
@@ -54,7 +54,7 @@ IndigoMolecule::IndigoMolecule() : IndigoBaseMolecule(MOLECULE)
 {
 }
 
-bool IndigoBaseMolecule::is(IndigoObject& object)
+bool IndigoBaseMolecule::is(const IndigoObject& object)
```

Comment:
There are some changes, useful, but not related to the subject of the pull-request/task.
It's better to extract them in a separate pr.

Reply:
The changes are related to the PR due to it requires them to avoid compile errors.
Other code is not affected.
But if you don't want them, I can rollback them and do workaround.

Reply:
OK. No problem.